gra_ceipla_zimno.py

```diff
@@ -0,0 +1,26 @@
+import random
+
+guest_num = random.randint(1, 51)
+counter = 0
+
+print('Zgadnij liczbę z zakresu 1-50: ')
+
+while True:
+    counter += 1
+    try:
+        temp = int(input('Podaj liczbę\n'))
+    except:
+        print('Proszę podać liczbę całkowitą')
+        continue
+
+    if temp == guest_num:
+        print("Zgadełeś")
+        break
```

Comment:
Można rozszerzyć funkcjonalność o zapytanie użytkownika czy chce zagrać jeszcze raz, w zależnośći od odpowiedzi zakończyć program lub zagrać jeszcze raz.

Reply:
Ok, ja dodałem dodatkowe warunki ale zdublowałem linijkę kodu losującą nową liczbę do zgadnięcia. Wiem, że można by było też dodać funkcję, ale przy tak krótkim skrypcie to raczej nie warto by było rozwiązywać to tak.